src/components/InvertedFlatList/BaseInvertedFlatList.js

```diff
@@ -2,8 +2,6 @@ import PropTypes from 'prop-types';
 import React, {forwardRef} from 'react';
 import FlatList from '@components/FlatList';
 
-const AUTOSCROLL_TO_TOP_THRESHOLD = 128;
```

Comment:
Do we actually want to remove this?

Reply:
`autoscrollToTopThreshold` will scroll down the list in the comment-linking process, so we need to ditch it.